tests-web/www/js/main.js

```diff
@@ -14,20 +14,24 @@ import {
   makeDotLayer,
   makeDebugLayer,
 } from "./layers.js";
+import {
+  makeSettingsControl,
+  makeLayerControl,
+  LayerGroup,
+} from "./controls.js";
 import init, { JsStreetNetwork } from "./osm2streets-js/osm2streets_js.js";
 
 await init();
 
 export class StreetExplorer {
   constructor(mapContainer) {
-    const [map, layerControl] = setupLeafletMap(mapContainer);
-    this.map = map;
-    this.layerControl = layerControl;
+    this.map = setupLeafletMap(mapContainer);
     this.currentTest = null;
     this.importSettings = {
       debugEachStep: false,
       dualCarriagewayExperiment: false,
     };
+    this.layers = makeLayerControl(this).addTo(this.map);
```

Comment:
Previously we had a `TestCase` own a list of layers, but the `LayerControl` live at the app level. That got confusing here. Now all of the layers in their groups are "owned" by the new layer control widget. That lives at the app level.